templates/vue/src/services/TapestryAPI.js

```diff
@@ -201,4 +204,10 @@ export default class {
     })
     return response
   }
+
+  async getAllContributors() {
+    const url = `${apiUrl}/tapestries/contributors?post_id=${this.postId}`
```

Comment:
Shouldn't this be ``${apiUrl}/tapestries/${this.postId}/contributors`` to be consistent with the rest of the endpoints?